packages/qgrid-core/src/pagination/pagination.let.js

```diff
@@ -2,6 +2,8 @@ export class PaginationLet {
 	constructor(plugin) {
 		const { model, observe } = plugin;
 
+		Object.defineProperty(this, 'current', { get: () => plugin.model.pagination().current });
```

Comment:
Please just do like this.plugin = plugin; consider not to use Object.defineProperty, we are on es6 :)